### PR TITLE
Add `event.relatedTarget` for mouseover and mouseout events

### PR DESCRIPTION
**Pull request by davidaurelio**
This adds a `relatedTarget` property to mouseover and mouseout event objects. It refers to the bonsai element that has been left or accessed.

### CODE REVIEW

File: example/library/movies/movie_list.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ movieList = {
```
Comment (klipstein) [inline example/library/movies/movie_list.js:72]: why do we have to maintain two movie lists?

File: src/renderer/svg/svg_event_handlers.js [hunk 2]
```diff
@@ -16,6 +19,40 @@ define([
...
   }
 
+  /**
+   * Returns the bonsai id of a DOM node
+   *
+   * @param {Node} node A DOM node
+   * @return {number} The bonsai id of the dom node or -1
+   */
+  function getBonsaiIdOf(node) {
+    var id = node && node.nodeType === ELEMENT_NODE && node.getAttribute('data-bs-id');
```
Comment (klipstein) [inline src/renderer/svg/svg_event_handlers.js:29]: On https://github.com/uxebu/bonsai/pull/171/files#L5R50 you use `bs-data-id` and here `data-bs-id`. Is that correct?

File: example/library/movies/movie_list.js (hunk 1 above)
Comment (davidaurelio) [inline example/library/movies/movie_list.js:72]: let me find out

File: src/renderer/svg/svg_event_handlers.js (hunk 2 above)
Comment (davidaurelio) [inline src/renderer/svg/svg_event_handlers.js:29]: of course not. Good catch. Thank you!

File: example/library/movies/movie_list.js (hunk 1 above)
Comment (davidaurelio) [inline example/library/movies/movie_list.js:72]: example/profiling uses movie_list.js, example/library the json file :(

72790428ae787d3fc8074b33358ab004c3f775d1 did not change all of them.

**Pull request merged**